src/App.styled.tsx

```diff
@@ -1,6 +1,8 @@
 import styled from 'styled-components';
+import { defaultTheme } from './themes/default';
 export const Styled = {
     App: styled.div`
+    ${defaultTheme};
```

Comment:
This is ok, but will be harder to read as you add more...
I'd go with :

```
const App = styled.div`...`;
export default {
   App,
}
```

Reply:
Changed to export the entire styled component as default.